examples/scripts/rloo/rloo.py

```diff
@@ -133,6 +133,6 @@ def tokenize(element):
     # Save and push to hub
     trainer.save_model(training_args.output_dir)
     if training_args.push_to_hub:
-        trainer.push_to_hub()
+        trainer.push_to_hub(dataset_name="trl-internal-testing/descriptiveness-sentiment-trl-style")
```

Comment:
Note to self: we should move these datasets that aren't strictly used for tests to `trl-lib`